test/framework/easyconfigs/toy-0.0-patches.eb

```diff
@@ -0,0 +1,32 @@
+name = 'toy'
```

Comment:
actually I don't wat to test with None, I'm testing easyblock code here, not easyconfig code.